frontend/public/style/_common.scss

```diff
@@ -202,6 +202,18 @@ input[type=number]::-webkit-outer-spin-button {
   background-color: #D64456;
 }
 
+.co-action-divider {
+  display: none;
+  @media(min-width: $screen-md-min) {
+    display: inline;
+    color: $color-pf-black-600;
+    padding-right: 10px;
+    &.no-padding {
```

Comment:
This is because btn-link has padding?

I'd rather have a class that removes padding for btn-link than a special case on the divider.

Reply:
Yes, `pairs-list` uses `<span>`'s whereas `resource-log` uses `<buttons>`

Reply:
How hard is it to switch pairs list to btn-link? Then we can remove the padding styles entirely and it fixes an accessibility issue.

@jcaianirh FYI

Reply:
> How hard is it to switch pairs list to btn-link? Then we can remove the padding styles entirely and it fixes an accessibility issue.

Switched pairs list to use `<button>`'s, consolidated scss classes.